tests/v2/test_wakunode.nim

```diff
@@ -1164,4 +1168,107 @@ asyncTest "Messages relaying fails with non-overlapping transports (TCP or Webso
     check:
       (await completionFut.withTimeout(5.seconds)) == false
     await node1.stop()
-    await node2.stop()
\ No newline at end of file
+    await node2.stop()
+
+asyncTest "Messages are relayed between nodes with multiple transports (TCP and secure Websockets)":
+    let
+      nodeKey1 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      node1 = WakuNode.new(nodeKey1, ValidIpAddress.init("0.0.0.0"),
+        bindPort = Port(60000), wsBindPort = Port(8000), wssEnabled = true, secureKey = KEY_PATH, secureCert = CERT_PATH)
+      nodeKey2 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+      node2 = WakuNode.new(nodeKey2, ValidIpAddress.init("0.0.0.0"),
+        bindPort = Port(60002))
+      pubSubTopic = "test"
+      contentTopic = ContentTopic("/waku/2/default-content/proto")
+      payload = "hello world".toBytes()
+      message = WakuMessage(payload: payload, contentTopic: contentTopic)
+
+    await node1.start()
+    node1.mountRelay(@[pubSubTopic])
+
+    await node2.start()
+    node2.mountRelay(@[pubSubTopic])
+
+    await node1.connectToNodes(@[node2.peerInfo.toRemotePeerInfo()])
+
+    var completionFut = newFuture[bool]()
+    proc relayHandler(topic: string, data: seq[byte]) {.async, gcsafe.} =
+      let msg = WakuMessage.init(data)
+      if msg.isOk():
+        let val = msg.value()
+        check:
+          topic == pubSubTopic
+          val.contentTopic == contentTopic
+          val.payload == payload
+      completionFut.complete(true)
+
+    node1.subscribe(pubSubTopic, relayHandler)
+    await sleepAsync(2000.millis)
+    
+    await node2.publish(pubSubTopic, message)
+    await sleepAsync(2000.millis)
+
+
+    check:
+      (await completionFut.withTimeout(5.seconds)) == true
+    await node1.stop()
+    await node2.stop()
+
+asyncTest "Messages fails with wrong key path":
+    let
+      nodeKey1 = crypto.PrivateKey.random(Secp256k1, rng[])[]
+  
+    expect IOError:
+      # gibberish
+      discard WakuNode.new(nodeKey1, ValidIpAddress.init("0.0.0.0"),
+        bindPort = Port(60000), wsBindPort = Port(8000), wssEnabled = true, secureKey = "../../waku/v2/node/key_dummy.txt")
```

Comment:
How is this different from key above?

Reply:
its failure case so , just bad path case.